experimental/iterators/test/python/dialects/iterators/dialect.py

```diff
@@ -98,3 +103,33 @@ def testEndToEnd():
   engine = ExecutionEngine(mod)
   # CHECK: (6)
   engine.invoke('main')
+
+
+@run
+# CHECK-LABEL: TEST: testEndToEndWithInput
+def testEndToEndWithInput():
+  mod = Module.parse('''
+      !struct_type = !llvm.struct<(i32,i64)>
+      func.func @main(%input: !tabular.tabular_view<i32,i64>)
+          attributes { llvm.emit_c_interface } {
+        %stream = "iterators.tabular_view_to_stream"(%input)
+          : (!tabular.tabular_view<i32,i64>)
+            -> !iterators.stream<!struct_type>
+        "iterators.sink"(%stream) : (!iterators.stream<!struct_type>) -> ()
+        return
+      }
+      ''')
+  pm = PassManager.parse(
+      'convert-iterators-to-llvm,convert-memref-to-llvm,convert-func-to-llvm,'
+      'reconcile-unrealized-casts,convert-scf-to-cf,convert-cf-to-llvm')
+  pm.run(mod)
+
+  data = np.array([(0, 3), (1, 4), (2, 5)], dtype=[('a', 'i4'), ('b', 'i8')])
```

Comment:
Plz comment the format here, as it is non-standard.

Reply:
Improved in latest commit.